recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake"
+    homepage = "https://embree.github.io/"
+    _source_folder = "source_subfolder"
+    _build_folder = "build_subfolder"
+    exports_sources = "CMakeLists.txt"
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "tbb/2019_u9"
+
+    options = {
+        "shared": [
+            True,
+            False
+        ],
+        "fPIC": [
+            True,
+            False
+        ],
+        "backface_culling": [
+            True,
+            False
+        ],
+        "ignore_invalid_rays": [
+            True,
+            False
+        ],
+        "ray_masking": [
+            True,
+            False
+        ]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "backface_culling": False,
+        "ignore_invalid_rays": False,
+        "ray_masking": False,
+        "tbb:tbbmalloc": True,
```

Comment:
As Embree requires this specific option and currently we don't have this package available, we will or remove this option, or provide such configuration as pre-built package.

Reply:
How would you suggest to do that ? Wouldn't that cause the Embree build to fail if that option is required ?

Reply:
@uilianries  Thanks for your comment, would you please provide any guidance of how this PR could proceed ? Is there any change could be made on my side ?  😃

Reply:
It's a good question. We have similar situation for header-only recipes. We need to provide some option for these cases. @danimtb WDYT ?

Reply:
Well I thought if the tbb package with that specific option is missing one can simply build it with `--build missing` setting.  That's how I did on my dev machine.

Reply:
@KaoCC does this package really need that option of tbb activated? Can you try removing `"tbb:tbbmalloc": True,` and see what happens? Thanks

Reply:
@danimtb  ... strange. I just tested it on two distinct machines and it seems like it will also compiled without the flag. I recalled the flag was a requirement. Anyway I removed it and pushed the commit again.